app/models/AllGainsSessionModel.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import models.gains._
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.crypto.EncryptedValue
+
+case class AllGainsSessionModel(allGains: Seq[PolicyCyaModel], gateway: Option[Boolean] = None) {
+  def toSubmissionModel: GainsSubmissionModel = {
+
+    def convertEmptyToOption[A](items: Seq[A]): Option[Seq[A]] = if (items.isEmpty) None else Some(items)
+
+    val life: Option[Seq[LifeInsuranceModel]] = convertEmptyToOption(allGains.filter(elem => elem.policyType.contains("Life Insurance")).map(cya =>
+      LifeInsuranceModel(
+        customerReference = cya.policyNumber,
+        event = cya.policyEvent,
+        gainAmount = cya.amountOfGain.get,
```

Comment:
Should we make https://github.com/hmrc/income-tax-additional-information/pull/24/files#diff-87d65a1ad0bbc84a0dea2a91fdd7e5301e49061a2d6a5a4f86109bcff55ddbe2R26 an `Option` to avoid using `.get` here?

Reply:
After a discussing this with Rob, and was advised that **_gainAmount_** will always need a value provided, so rather than change the attribute to be an option, I could provide a zero value to it - `gainAmount = cya.amountOfGain.getOrElse(BigDecimal(0))`